test_posting.py

```diff
@@ -20,3 +20,9 @@
 request = requests.post(url, json=obj, headers=headers)
 
 print(json.dumps(request.json(), indent=4))
+
+url = "http://127.0.0.1:5000/feed"
+params = {"session-id": "6f3225f5-5fd8-4f58-a505-670592e46b9b"}
+request = requests.post(url, params=params)
+
+print(json.dumps(request.json(), indent=4))
```

Comment:
The format is likely needed to be changed to fit what the frontend wants. But we can figure those details out I guess after the first merge? @NickCallaghan